src/arith_uint256.h

```diff
@@ -196,6 +197,7 @@ class base_uint
         return ret;
     }
 
+    /** Numeric ordering (unlike \ref base_blob::Compare) */
```

Comment:
no strong preference either way, just noticing that in other cases it's rather called https://en.wikipedia.org/wiki/Natural_sort_order